backend/app/usecase/repository/url.go

```diff
@@ -1,11 +1,17 @@
 package repository
 
-import "github.com/short-d/short/app/entity"
+import (
+	"time"
+
+	"github.com/short-d/short/app/entity"
+)
 
 // URL accesses urls from storage, such as database.
 type URL interface {
 	IsAliasExist(alias string) (bool, error)
 	GetByAlias(alias string) (entity.URL, error)
+	// TODO change to CreateURL
```

Comment:
Please find our TODO format here: [s/swe-onboarding](http://s/swe-onboarding)

Reply:
Will create the issue and update the TODO to match short format.